airflow/task/task_runner/standard_task_runner.py

```diff
@@ -186,3 +193,20 @@ def get_process_pid(self) -> int:
         if self.process is None:
             raise RuntimeError("Process is not started yet")
         return self.process.pid
+
+    def _read_task_utilization(self):
+        dag_id = self._task_instance.dag_id
+        task_id = self._task_instance.task_id
+
+        while True:
+            try:
```

Comment:
And also thin we better wrap loop in try...except

```python
try:
    while True:
        # do something
except ...:
    return
```

Reply:
Done